testing/run_tests.py

```diff
@@ -75,19 +75,23 @@ def run_cmd(
   print(f'Running command "{command_string}"')
 
   start_time = time.time()
-  collect_output = forbidden_output or allowed_failure_output
-  stdout_pipe = sys.stdout if not collect_output else subprocess.PIPE
-  stderr_pipe = sys.stderr if not collect_output else subprocess.PIPE
 
   process = subprocess.Popen(
       cmd,
-      stdout=stdout_pipe,
-      stderr=stderr_pipe,
+      stdout=subprocess.PIPE,
```

Comment:
This change allows us to stream the output to stdout but also capture it for inspection.  The tests take a long enough time, it's a pain not seeing the streaming results.